packages/hardhat-core/test/internal/cli/autocomplete.ts

```diff
@@ -33,18 +34,22 @@ const coreTasks = [
 ];
 
 const coreParams = [
-  "--network",
-  "--show-stack-traces",
-  "--version",
-  "--help",
-  "--emoji",
   "--config",
+  "--emoji",
+  "--help",
   "--max-memory",
+  "--network",
+  "--show-stack-traces",
   "--tsconfig",
   "--verbose",
+  "--version",
 ];
 
-describe("autocomplete", () => {
+describe("autocomplete", function () {
+  if (os.type() === "Windows_NT") {
```

Comment:
I did this mainly to avoid having to use `path.sep` in a lot of places, since bash/zsh/fish don't work on Windows, right? (And if they are emulated or something, they use forwards slashes AFAIK.)